### PR TITLE
Object null reference fix

### PR DESCRIPTION
**Pull request by apraovjr**
## Description

Fix for object not set to instance of object in remove eventhubnamespace.
Issue: #10634

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/EventHub/EventHub.Test/ScenarioTests/NamespaceTests.ps1 [hunk 1]
```diff
@@ -177,6 +177,7 @@ function NamespaceAuthTests
     
     Write-Debug " Delete namespaces"
     Remove-AzEventHubNamespace -ResourceGroup $resourceGroupName -Name $namespaceName
+	Remove-AzEventHubNamespace -ResourceId $resultkafka.Id
```
Comment (msJinLei) [inline src/EventHub/EventHub.Test/ScenarioTests/NamespaceTests.ps1]: ident seems to be incorrect.

File: src/EventHub/EventHub/Cmdlets/AzureEventHubsCmdletBase.cs [hunk 2]
```diff
@@ -196,23 +196,26 @@ public class LocalResourceIdentifier : ResourceIdentifier
...
+                        break;
+                    case 4:
+                        ParentResource = tokens[1];
+                        ParentResource1 = tokens[3];
+                        break;
+                    case 6:
+                        ParentResource = tokens[1];
+                        ParentResource1 = tokens[3];
+                        ParentResource2 = tokens[5];
+                        break;
```
Comment (msJinLei) [inline src/EventHub/EventHub/Cmdlets/AzureEventHubsCmdletBase.cs]: Please add a default logic here.

**Pull request merged**